game/fs.hh

```diff
@@ -99,3 +99,12 @@ Paths const& getPaths();  ///< Get the data file search path
 Paths getThemePaths();  ///< Get the data/theme file search path (includes current and default themes in addition to data folders)
 Paths getPathsConfig(std::string const& confOption);  ///< Return expanded list of paths specified by a path config option
 
+template <>
+class std::hash<fs::path>
```

Comment:
std::hash seems to be a struct (AFAIR, windows care about struct/class), and this would prevent the need for `public:` ; may also be noexcept

Reply:
Changed so it's a struct instead of class and added the noexcept